apps/testing/integration-suite/scripts/ci-test.sh

```
@@ -23,16 +23,23 @@ echo "Integration Suite - CI Test Runner"
 echo "==================================="
 echo ""
 
+# Load .env.local if it exists and AGENTUITY_SDK_KEY is not already set
+if [ -z "$AGENTUITY_SDK_KEY" ] && [ -f "$APP_DIR/.env.local" ]; then
+	echo -e "${YELLOW}ℹ${NC} Loading environment from .env.local for local development"
+	export $(grep -v '^#' "$APP_DIR/.env.local" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)